### PR TITLE
re-enable guard-rspec since we are on ruby 2.5

### PR DESCRIPTION
**Pull request by kencheeto**
### Description
guard-rspec is nice for adding `focus: true` to tests then running `bundle exec guard` to automatically rerun tests when watched files change.

### Acceptance Criteria
- [x] `bundle exec guard` will rerun tests on file changes

### Testing Plan
N/A

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -119,12 +119,12 @@ group :stubbed, :test, :development, :demo do
   gem "bundler-audit"
 
   # Testing tools
+  gem "capybara"
```
Comment (kencheeto) [inline Gemfile]: reorganizing these bc the commented `guard-rspec` line was signaling a new group of gems to rubocop, which enforces alpha ordering of gems within a 'group' as delimited by comments or newlines.

File: Guardfile [hunk 2]
```diff
@@ -2,8 +2,8 @@
 # More info at https://github.com/guard/guard#readme
 
 ## Uncomment and set this to only include directories you want to watch
-# directories %w(app lib config test spec features) \
-#  .select{|d| Dir.exists?(d) ? d : UI.warning("Directory #{d} does not exist")}
+directories(%w[app lib config spec])\
```
Comment (kencheeto) [inline Guardfile]: without this line, there's an error that some node_modules directory is already being watched. I don't know the `directories` call is required (shouldn't guard only watch the files specified in `watch` clauses?) but it does get rid of the error.

**Comment by pkarman** [comment]
Maybe worth a mention in the README?

**Comment by kencheeto** [comment]
good tip, thanks @pkarman !

**Pull request merged**